assets/meili-love.md

```diff
@@ -0,0 +1,61 @@
+# Love from our users
+
+- 20/03/2020: [love the performance and simplicity of MeiliSearch](https://github.com/meilisearch/MeiliSearch/issues/535#issue-585176606)
```

Comment:
My bad, I should have mention it, but can you put the date YYYY-MM-DD :D

It make more sense when you order by date ASC 😬